dronegen/tag.go

```diff
@@ -66,6 +66,27 @@ func tagPipelines() []pipeline {
 		},
 	}))
 
+	ps = append(ps, ghaBuildPipeline(ghaBuildType{
```

Comment:
If you don't mind giving this another week or so, I'll have the tag build pipelines off drone completely, real soon now (I hope to start raising the master PRs tomorrow). It would be good to avoid updating drone when it already has the knife in its heart and is slowly bleeding out. If you have a timeline you need to meet for this, let me know and we'll figure something out.